src/mind.rs

```diff
@@ -134,6 +135,9 @@ pub fn α_β_negamax_search(world: WorldState,
         if experienced_α >= β {
             break;
         }
+        if (20000.0 - experienced_α.abs()).abs() < 1000.0 {
```

Comment:
yeah, it took me a non-trivial amount of code-reading & understanding the actual algorithm to find the correct place for this superficially tiny fix. α-β search is surprisingly difficult to understand given the small amount of code required to implement it!